src/AccessibilityInsights.Core/Bases/A11yElement.cs

```diff
@@ -88,6 +109,9 @@ public string Culture
             }
         }
 
+        /// <summary>
+        /// a permanent unique identifier for the specific element, or sometimes the type of element
```

Comment:
>permanent unique identifier [](start = 14, length = 27)

Is it worth calling out that the uniqueness is not global, but only unique within siblings in its container?

Reply:
```suggestion
        /// a permanent unique identifier among siblings for the specific element, or sometimes the type of element
```